### DISCUSSION TITLE
create your access token

### DISCUSSION THREAD

**Discussion by Bagfish** [pyannote/pyannote-audio#1121] [Q&A] [answered]
Could not download 'pyannote/voice-activity-detection' pipeline.
It might be because the pipeline is private or gated so make
sure to authenticate. Visit https://hf.co/settings/tokens to
create your access token and retry with:

   >>> Pipeline.from_pretrained('pyannote/voice-activity-detection',
   ...                          use_auth_token=YOUR_AUTH_TOKEN)

If this still does not work, it might be because the pipeline is gated:
visit https://hf.co/pyannote/voice-activity-detection to accept the user conditions.

**Comment by mezaros** [comment]
Doesn't work. Coming from the same IP, I've gone through the gate, and created a token. Same error as before.

Does anyone know why this is necessary? Why can't we just download and access the model locally? This feels fragile and so far has stopped all of my tests in their tracks.

**Comment by hbredin** [comment, marked as answer]
I have just updated the [FAQ](https://github.com/pyannote/pyannote-audio/blob/develop/FAQ.md#can-i-use-gated-models-and-pipelines-offline) with instructions on how to use pretrained models and pipelines offline.
